brave/browser-laptop
# Don't call for IPC actions for SHORTCUT_ACTIVE_FRAME_FORWARD and SHORTCUT_ACTIVE_FRAME_BACK

Auditors @bbondy, @luixxiul
Close #11346

Manual Test plan:

1. in the same tab, visit different 3 websites
2. go back to the second site you visited
3. now that you have both `previous` and `forward` buttons enabled in navigation bar,
toggle `back`/`forward`, they should work
4. go back to position where you have `previous` and `forward` buttons enabled in navigation bar
5. shortcut `CmdOrCtrl+[` should go back and `CmdOrCtrl+]` should go forward
6. shortcut `cmd+left` should go back and `cmd+right` should go forward
7. in macOS, two-finger swipe right/left. Should navigate back/forward
8. inside the `webview`, open context menu and `back`/`forward` options should work
9. go to top-level menu -> history -> `back`/`forward` actions should work

Test plan:

```
npm run test -- --grep="back-forward state"
# other back/forward actions are handled by several unit tests
# which should all pass too
npm run unittest
```
